docs/sql/statements/copy-from.rst

```diff
@@ -114,6 +116,10 @@ URI
 A string literal or array of string literals containing URIs. Each URI must be
 formatted according to the `URI Scheme`_.
 
+The ``COPY FROM`` statement allows ``URIs`` with wildcards to conveniently match all
```

Comment:
@norosa Hello Nomi, I did a little bit of doc refactoring since now URIs with folder level wildcard is allowed for both S3 and file schemes. I would appreciate if you could have quick look. Thanks!